### ISSUE TITLE
Document conflicts with VSCode-R  

### ISSUE THREAD

**Issue by jpgoldberg** [TianyiShi2001/rmarkdown-vscode#17]
The Rmarkdown extension does not play nice with the [R extension](https://marketplace.visualstudio.com/items?itemName=Ikuyadeu.r). I would recommend that this be documented upfront.

It might be useful to look at what this extension does which the R extension doesn't and find a way to just enable those features to avoid a conflict.

**Comment by TeddyHuang-00** [comment]
True, I found the same problem since I got this extension installed, for I can no longer use the code formatter in vscode-R which used to play nice and in my opinion is very important that simply cannot be ditched.
